README.md

```diff
@@ -71,7 +70,6 @@ Support Us on [Liberapay][17] or [Bountysource][18]
 
 [1]: https://cuberite.org/news/#subscribe
 [2]: https://cuberite.org/
-[3]: https://builds.cuberite.org/
```

Comment:
Ideally change the numbers too.

Reply:
Way too many unrelated changes from that